bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -2954,6 +2966,10 @@
 		<label>Enable Motion Alarm</label>
 		<description>By using this feature you can stop the camera from sending e-mails when you are actually home.</description>
 		<category>Motion</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Motion</tag>
```

Comment:
I'm not sure about this combination - I don't think you can control motion using this channel type?

Reply:
Status.alarm or similar (on my phone, no access to all lists now)